pybamm/models/submodels/electrolyte_current.py

```diff
@@ -484,9 +484,9 @@ def get_post_processed(self, delta_phi_n, delta_phi_p, i_e_n, i_e_p, c_e, eps=No
         # import parameters and spatial variables
         param = self.set_of_parameters
         i_cell = param.current_with_time
-        x_n = pybamm.standard_spatial_vars.x_n
-        x_s = pybamm.standard_spatial_vars.x_s
-        x_p = pybamm.standard_spatial_vars.x_p
+        # x_n = pybamm.standard_spatial_vars.x_n
```

Comment:
I think you might have missed either uncommenting or deleting these lines

Reply:
Somewhat messily fixed these lines in a different PR, so will merge this now